app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java

```diff
@@ -1058,7 +1062,7 @@ public void onAddNutriScorePromptClick() {
         //adds the information about the prompt when navigating the user to the edit the product
         intent.putExtra("modify_category_prompt", showCategoryPrompt);
         intent.putExtra("modify_nutrition_prompt", showNutrientPrompt);
-        startActivity(intent);
+        startActivityForResult(intent, EDIT_REQUEST_CODE);
```

Comment:
The activity must be started with startActivityForResult(), since we refresh the visibility of the views by calling refreshView() in the onActivityResult() method.